input/fsh/modules/files.fsh

```diff
@@ -139,4 +161,24 @@ Description: "Information about a file related to a research participant"
 * description ^short = "A description of the file"
 * type 0..1 /*File Type*/
 * type from edam-ontology-terms (extensible) 
-* type ^short = "The type of data contained in this file."
\ No newline at end of file
+* type ^short = "The type of data contained in this file."
+
+Profile: NcpiDRSFile
+Parent: NcpiFile
+Id: ncpi-drs-file
+Title: "NCPI DRS File"
+Description: "Information about a DRS file related to a research participant"
+* ^version = "0.0.1"
+* ^status = #draft
+* content ^slicing.discriminator.type = #pattern
+* content ^slicing.discriminator.path = "code"
+* content ^slicing.rules = #openAtEnd
+* content ^slicing.ordered = false
+* content ^slicing.description = "Slicing pattern to make content.attachment require a DRS file type and allow other file types"
+* content contains
+  DRS 1..1 and
+  Other 0..*
+* content[DRS].attachment only DRSAttachment
+//* content[DRS].profile only 
+* content[Other].attachment only Attachment

```

Comment:
I believe you can leave the Other slice out. I believe the open at end is sufficient for allowing other types of attachments in addition to the DRS one(s).